Chapter 13 (#17)
LaunchCodeEducation/csharp-web-dev-curriculum: content/razor-views/_index.md

---
title: "Razor Views"
date: 2023-02-09T12:48:24-06:00
draft: false
weight: 13
originalAuthor: Courtney Frey # to be set by page creator
originalAuthorGitHub: speudusa # to be set by page creator
reviewer: Kimberly Horan # to be set by the page reviewer
reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
lastEditor: # update any time edits are made after review
lastEditorGitHub: # update any time edits are made after review
lastMod: # UPDATE ANY TIME CHANGES ARE MADE
---

## Learning Objectives 

> After completing this chapter you should be able to 

Reviewer: ```suggestion
By the end of this chapter, you should understand:
```

> 1. Understand the benefits of using templates
> 1. Create a template using a Razor View in an MVC application
> 1. How to add iteration, conditionals, and forms to a template
> 1. How to link a template to static resources like CSS stylesheets, scripts, images, etc.
> 1. How to render a template using ASP.NET controllers

Reviewer: ```suggestion
1. The benefits of using templates
1. How to create a template using a Razor View in an MVC application
1. How to add iteration, conditionals, and forms to a template
1. How to link a template to static resources like CSS stylesheets, scripts, images, etc.
1. How to render a template using ASP.NET controllers
```
